features/system-report/src/.gitignore

```diff
@@ -1 +0,0 @@
-META-INF
```

Comment:
Are these files regenerated during each build or was ```src/META-INF``` added to this .gitignore by mistake?